spec/features/collections/breadcrumbs_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'Collections breadcrumbs' do
+  before do
+    login
+    ingest_response, @concept_response = publish_collection_draft
```

Comment:
not necessary to make this change, but would it reduce a bit of test time to visit one of the LARC or SEDAC collections so we don't have to ingest one for this?